### PR TITLE
Issue 44: Add --private flag when running scan 

### PR DESCRIPTION
**Pull request by tombui99**
Added --private flag to run scan privately in Docker

### CODE REVIEW

File: docker/index.js [hunk 1]
```diff
@@ -248,6 +254,10 @@ const processAndUpload = async (
 		);
 	}
 
+	if (args.private) {
```
Comment (nvhoanganh) [inline docker/index.js:257]: this can move down directly to line 309

File: docker/index.js [hunk 2]
```diff
@@ -296,6 +306,7 @@ const processAndUpload = async (
 				code: codeAuditor,
 				htmlIssuesSummary,
 				htmlIssues,
+				isPrivate
```
Comment (nvhoanganh) [inline docker/index.js:309]: this can be `isPrivate: args.private`

**Review by nvhoanganh — CHANGES_REQUESTED** [review]

File: docker/index.js (hunk 1 above)
Comment (tombui99) [reply docker/index.js]: Done - moved to line 309

File: docker/index.js (hunk 2 above)
Comment (tombui99) [reply docker/index.js]: Done - Updated line 309

File: docker/index.js [hunk 3]
```diff
@@ -296,6 +302,7 @@ const processAndUpload = async (
 				code: codeAuditor,
 				htmlIssuesSummary,
 				htmlIssues,
+				isPrivate: args.private ? true : false
```
Comment (nvhoanganh) [inline docker/index.js:305]: args.private should already be a boolean, don't need this

**Review by nvhoanganh — CHANGES_REQUESTED** [review]

File: docker/index.js (hunk 3 above)
Comment (tombui99) [reply docker/index.js]: Done - Updated

**Review by nvhoanganh — APPROVED** [review]

**Pull request merged**